### ISSUE TITLE
result has no bbox 

### ISSUE THREAD

**Issue by decajcd** [hasanirtiza/Pedestron#163]
I run 'python tools/train.py configs/elephant/cityperson/cascade_hrnet.py'  to train,and in configs/elephant/cityperson/cascade_hrnet.py,img_scale=[(1216, 608), (2048, 1024)] is modified to [(608, 304), (1024, 512)],load_from='epoch_5.pth'.After one round of training,when i run python demo.py,the result has no bbox,just oral image

**Comment by hasanirtiza** [comment]
With this information cant really comment on anything. How was your training log. Did you first simply try running the epoch5.pth  on your images ?

**Comment by decajcd** [comment]
> With this information cant really comment on anything. How was your training log. Did you first simply try running the epoch5.pth on your images ?
epoch5.pth has bbox

**Comment by decajcd** [comment]
> With this information cant really comment on anything. How was your training log. Did you first simply try running the epoch5.pth on your images ?
when I  trainthe model,the acc is about 98%,but when I run demo,it has no box

**Comment by hasanirtiza** [comment]
Its somewhat of a strange behavior, may be let it run for few epochs. Secondly, you can also try to reduce the learning rate. Lastly, as a sanity check, make sure your annotations are correct.

**Comment by decajcd** [comment]
> Its somewhat of a strange behavior, may be let it run for few epochs. Secondly, you can also try to reduce the learning rate. Lastly, as a sanity check, make sure your annotations are correct.

After modify img_scale,how to change bbox location?
